src/test/java/uk/gov/pay/ledger/util/fixture/QueueEventFixtureUtil.java

```diff
@@ -39,8 +40,10 @@ public static PactDslJsonBody getAsPact(String eventType, ZonedDateTime eventDat
                                      String parentResourceExternalId, ResourceType resourceType, String eventData) {
         PactDslJsonBody eventDetails = new PactDslJsonBody();
 
+        DateTimeFormatter formatToMicroseconds = DateTimeFormatter.ofPattern("uuuu-MM-dd'T'HH:mm:ss.SSSSSSX");
```

Comment:
Is there a reason this couldn't use the static https://github.com/alphagov/pay-ledger/blob/f76e13cdf4f369e298f71a80f148355d5d4d6bf5/src/main/java/uk/gov/pay/ledger/event/model/serializer/MicrosecondPrecisionDateTimeSerializer.java linked in the description?